docker-compose-generator/docker-fragments/opt-add-tor-relay.yml

```diff
@@ -0,0 +1,16 @@
+version: "3"
+services:
+  tor-relay:
```

Comment:
is it possible to use our image instead? because our image works on all architecture.

Reply:
@Kukks actually you should use the same **container**! Why the need of two tor instances?

The config of the main tor image is actually templated so you can customize like nginx.

Reply:
hmm, what is the templating language used and how can I get access to env vars in there?

This would change completely the new feature: instead of a fragment we need a way to modify the template

Reply:
Yeah that mean the new fragment would not even be needed, just setting the right env variable

Reply:
done